common:
    lid_topic:  "/hesai/pandar"
    imu_topic:  "/mavros/imu/data_raw"

preprocess:
    lidar_type: 5                # Hesai PandarXT
    scan_line: 32
    blind: 3
    feature_extract_en: false

initialization:
    cut_frame_num: 4 # must be positive integer
    orig_odom_freq: 10
    mean_acc_norm: 9.805
    online_refine_time: 15.0
    data_accum_length: 300
    Rot_LI_cov: [ 0.00005, 0.00005, 0.00005 ]
    Trans_LI_cov: [ 0.00001, 0.00001, 0.00000001 ]

mapping:
    filter_size_surf: 0.05
    filter_size_map: 0.15
    gyr_cov: 20
    acc_cov: 2
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    det_range: 120.0

publish:
    path_en:  true
    scan_publish_en:  true       # false: close all the point cloud output
    dense_publish_en: true       # false: low down the points number in a global-frame point clouds scan.
    scan_bodyframe_pub_en: true  # true: output the point cloud scans in IMU-body-frame

pcd_save:
    pcd_save_en: false
    interval: -1                 # how many LiDAR frames saved in each pcd file; 
                                 # -1 : all frames will be saved in ONE pcd file, may lead to memory crash when having too much frames.
